collect_app/build.gradle

```diff
@@ -10,6 +10,29 @@ com.android.ddmlib.DdmPreferences.setTimeOut(60000)
 ant.condition(property: "os", value: "windows") { os(family: "windows") }
```

Comment:
I don't believe these two lines are used since the property is directly accessed in the switch statements below.